### ISSUE TITLE
Format only being applied to local default user

### ISSUE THREAD

**Issue by OddSquirrel** [bhilburn/powerlevel9k#1298]
Hi everybody,

I just installed iTerm2, oh-my-zsh and powerlevel9k as well as a nerd font. So far, everything's working beautifully, except for the formatting, which only applies to the default user. Bit hard to explain, but as soon as I change my "persona" via switching to root or connect to another server via ssh the entire prompt formatting is gone. This is what it looks like:

![2019-06-03_18-32-17](https://user-images.githubusercontent.com/39886438/58818810-07906800-862f-11e9-87ea-fc17831df0e9.jpg)

Does anybody have an idea what I may be doing wrong?

**Comment by romkatv** [comment]
`sudo su` doesn't load your `~/.zshrc`. Use `sudo zsh` instead.

**Comment by OddSquirrel** [comment]
Thanks @romkatv! I wasn't aware of that, I thought sudo su is a thing, no matter what shell (like I wouldn't type sudo bash in bash if that makes sense). So it did indeed work and format my prompt. Unfortunately, all my ssh connection aliases no longer work that way. :(

**Comment by romkatv** [comment]
You can use `sudo $SHELL` as a shell-agnostic command.

**Comment by OddSquirrel** [comment]
That is great to know, thanks much!

Would you have an idea why it won't format my ssh prompt, though?

**Comment by romkatv** [comment]
> Would you have an idea why it won't format my ssh prompt, though?

It should work as long as you have your theme installed on the remote machine to which you are connecting. Do you?

**Comment by OddSquirrel** [comment]
> It should work as long as you have your theme installed on the remote machine to which you are connecting. Do you?

Well no, I mean I can't install all kinds of mods on other people's systems of course. Are you telling me my terminal needs another machine to serve prompt formatting back? If so, I'm completely lost now. I mean I'm literally telling my system that I'm SSHing into a remote machine and it can't work with that to change colors and backgrounds without the remote machine telling it the exact same thing? Seriously?

**Comment by romkatv** [comment]
> I'm literally telling my system that I'm SSHing into a remote machine and it can't work with that to change colors and backgrounds without the remote machine telling it the exact same thing? Seriously?

Yes. Don't look at me, it wasn't my idea. Your terminal is a glorified teletypewriter (a.k.a. TTY). These things go back to the 19th century, with only cosmetic improvements since then.

**Comment by OddSquirrel** [comment]
Oh, no worries, you're not responsible. I almost fell off my chair, though. What a backwards way of doing things! As you already said, those terminals helped Columbus discovering the Americas, so we get to take it or leave it I reckon. Thanks again for your help!
